WoWsShipBuilder.Core/DataContainers/Projectiles/ShellDataContainer.cs

```diff
@@ -9,6 +9,7 @@
 using WoWsShipBuilder.DataElements.DataElementAttributes;
 using WoWsShipBuilder.DataElements.DataElements;
 using WoWsShipBuilder.DataStructures;
+using WoWsShipBuilder.DataStructures.Projectile;
 
 namespace WoWsShipBuilder.Core.DataContainers
 {
```

Comment:
Which modifiers are the problematic ones for the TODO in the AP shell case?

Reply:
No idea why this was there but since it has been there for a year now, it does not seem to be an issue. Removed the todo